ui/src/components/AlarmFilters.vue

```diff
@@ -93,6 +111,10 @@ const updateList = (severities: string[]) => {
 				<FeatherIcon :icon="CheckCircle" aria-hidden="true" class="icon ack" />
 				<span>Acknowledge</span>
 			</FeatherButton>
+			<FeatherButton class="acction-btn" @click="() => handleRemoveAlarm()">
+				<FeatherIcon :icon="Delete" aria-hidden="true" class="icon remove" />
```

Comment:
If the goal with `aria-hidden` is to remove the icon from the accessibility api, I believe the FeatherIcon does that by default, unless a title is specified. Pretty sure it's safe to remove them